### PR TITLE
Otters security fixes 2.6

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: components/function-runtimes/python39/Dockerfile [hunk 1]
```diff
@@ -48,6 +48,8 @@ RUN set -ex \
 	&& tar -xJC /usr/src/python --strip-components=1 -f python.tar.xz \
 	&& rm python.tar.xz \
 	\
+	&& apk add --no-chacje --repository=https://dl-cdn.alpinelinux.org/alpine/edge/main \
```
Comment (pPrecel) [inline components/function-runtimes/python39/Dockerfile:51]:
```suggestion
	&& apk add --no-cache --repository=https://dl-cdn.alpinelinux.org/alpine/edge/main \
```

File: components/function-runtimes/python39/Dockerfile [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # We use this code to manually use newest alpine and nodejs.
 # Use official build when it will be available.
 
-FROM alpine:3.15.4
+FROM alpine:3.16.2
```
Comment (dbadura) [inline components/function-runtimes/python39/Dockerfile:10]: We can use sha image:
```
# image base on alpine:3.16.2
FROM alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad
```

**Review by dbadura — CHANGES_REQUESTED** [review]
Bump images

**Review by pPrecel — DISMISSED** [review]

File: components/function-runtimes/python39/Dockerfile (hunk 2 above)
Comment (m00g3n) [reply components/function-runtimes/python39/Dockerfile]: lets change this, thx

**Review by pPrecel — DISMISSED** [review]

**Review by pPrecel — DISMISSED** [review]

**Review by dbadura — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Pull request merged**